plugins/scaffolder-backend/src/scaffolder/actions/builtin/helpers.ts

```diff
@@ -251,3 +251,7 @@ export function getGitCommitMessage(
 export function entityRefToName(name: string): string {
   return name.replace(/^.*[:/]/g, '');
 }
+
+export function printGitlabError(error: any): string {
```

Comment:
Is this just a `gitlab` thing? Or can it be called something more generic? I'd rather move this to the actions folder rather than in here if we can.

Reply:
it is gitlab specific, moved it to the gitlab.ts action file